base/features/spock/skeleton/maven-build/pom.xml

```diff
@@ -3,6 +3,8 @@
     <!-- JUnit 5 requires Surefire and Failsafe version 2.22.0 or higher -->
     <maven-surefire-plugin.version>2.22.2</maven-surefire-plugin.version>
     <maven-failsafe-plugin.version>2.22.2</maven-failsafe-plugin.version>
+    <junit-platform.version>1.2.0</junit-platform.version>
+    <jupiter.version>5.2.0</jupiter.version>    
```

Comment:
Can we move these hard coded versions into the BOM?

Reply:
Probably. How will they end up being here?

Reply:
PR to core to modify https://github.com/micronaut-projects/micronaut-core/blob/1.3.x/build.gradle#L260